dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -231,15 +224,19 @@ def is_primitive_type(x):
 
         return False, 'cpu', data
 
-    def _create_subpipeline(self, op_wrapper, inputs, **kwargs):
+    def _create_subpipeline(self, op_wrapper, inputs, kwargs):
```

Comment:
When you are adjusting the external_source, can you also add a comment that we store the classification of the data as batch in the first iter. Here or by the call site, I don't know which one is better.

Reply:
done